resources/testharness.js

```diff
@@ -2818,6 +2825,13 @@
         this.done();
     };
 
+    /**
+     * @func
+     *
+     * Manually set the test status to ``TIMEOUT``.
+     *
+     * Synonym for `Test.timeout <#Test.timeout>`_.
```

Comment:
Suggesting a word I see used more often about APIs:

```suggestion
     * Alias for `Test.timeout <#Test.timeout>`_.
```